### PR TITLE
fix failure for -maccel=NONE

### PR DESCRIPTION
**Pull request by AlexandreEichenberger**
My understanding is that it is valid to request a compilation with `--maccel=NONE`. Currently it fails because of pass registration failure.

This is fixed here.  Previously, we relied on `maccel.size()>0` imply that we have a valid accelerator. This is not true when `NONE` is added to `maccel`. This PR fixes that.

Signed-off-by: Alexandre Eichenberger <alexe@us.ibm.com>

### CODE REVIEW

File: src/Compiler/CompilerUtils.cpp [hunk 1]
```diff
@@ -752,7 +752,8 @@ void setupModule(mlir::OwningOpRef<ModuleOp> &module,
       std::string accelStr = accel->getName() + "-0x" + versionNumber.str();
       accels.emplace_back(StringAttr::get(&context, accelStr));
     }
-    moduleOp.setAttr("onnx-mlir.accels", ArrayAttr::get(&context, accels));
+    if (accels.size())
```
[the remark this answers is not in the document]
Comment (AlexandreEichenberger) [reply src/Compiler/CompilerUtils.cpp:755]: good suggestion, being done now.

**Comment by AlexandreEichenberger** [comment]
@jenkins-droid test this please

**Comment by AlexandreEichenberger** [comment]
@etiotto I believe I resolved all the issues you raised.

**Review by etiotto — APPROVED** [review]
LGTM

**Pull request merged**